packages/core/src/actions/getOpenOrders.ts

```diff
@@ -1,25 +1,32 @@
 import type { Config } from '../createConfig.js'
-import type { OrderMetadata } from '../types/order.js'
+import type { OpenOrder } from '../types/order.js'
 import { ADMIN_OPEN_ORDERS_ROUTE } from '../constants.js'
 import { getRelayerWithAdmin } from '../utils/http.js'
 import { BaseError, type BaseErrorType } from '../errors/base.js'
 
-export type GetOpenOrdersReturnType = Map<string, OrderMetadata>
+export type GetOpenOrdersParams = {
+  matchingPool?: string
+}
+
+export type GetOpenOrdersReturnType = Map<string, OpenOrder>
 
 export type GetOpenOrdersErrorType = BaseErrorType
 
 export async function getOpenOrders(
   config: Config,
+  parameters: GetOpenOrdersParams,
 ): Promise<GetOpenOrdersReturnType> {
   const { getRelayerBaseUrl } = config
 
-  const res = await getRelayerWithAdmin(
-    config,
-    getRelayerBaseUrl(ADMIN_OPEN_ORDERS_ROUTE),
-  )
+  let url = getRelayerBaseUrl(ADMIN_OPEN_ORDERS_ROUTE)
+  if (parameters.matchingPool) {
+    url += `?matching_pool=${parameters.matchingPool}`
```

Comment:
Probably better pattern to use the URL constructor and set search parameters that way for example:
```
if (parameters.matchingPool) {
  const temp = new URL(url)
  temp.searchParams.set('matching_pool', parameters.matchingPool)
  url = temp.toString()
}
```